examples/weather/dlwp_healpix_coupled/README.md

```diff
@@ -0,0 +1,16 @@
+# Deep Learning Weather Prediction (DLWP) model for weather forecasting
+
+This example is an implementation of the
```

Comment:
This looks like the regular DLWP healpix readme. This should be updated to one that's more specific to the coupled model

Reply:
Updated REDAME for DLWP healpix coupled